Source/SteveSloth/Public/MyEnemy.h

```diff
@@ -0,0 +1,28 @@
+// Fill out your copyright notice in the Description page of Project Settings.
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "GameFramework/Character.h"
+#include "MyEnemy.generated.h"
+
+/**
+ * 
+ */
+class STEVESLOTH_API MyEnemy
+{
+public:
+	float MaxHealth;
+	float CurrentHealth;
+	int Damage;
+	enum MovementState {Idle, Patrolling, Chasing, Fleeing, Walking};
```

Comment:
Enums are not variables. Cannot be instantiated this way
Instantiate the Enum outside the Class at the top.